pdns/pdns_recursor.cc

```diff
@@ -2566,6 +2567,9 @@ int serviceMain(int argc, char*argv[])
 
   showProductVersion();
   seedRandom(::arg()["entropy-source"]);
+  openssl_thread_setup();
+  openssl_seed();
```

Comment:
We should call `openssl_seed()` after `fork()`, if I'm not mistaken: https://wiki.openssl.org/index.php/Random_fork-safety